dali_tf_plugin/dali_compile_flags.py

```diff
@@ -14,22 +15,22 @@ def get_module_path(module_name):
             break
     return module_path
 
+
 def get_dali_build_flags():
     dali_cflags = ''
     dali_lflags = ''
     dali_include_flags = ''
     try:
         import nvidia.dali.sysconfig as dali_sc
-        dali_lib_path = dali_sc.get_lib_dir()
-        dali_include_flags=" ".join(dali_sc.get_include_flags())
-        dali_cflags=" ".join(dali_sc.get_compile_flags())
-        dali_lflags=" ".join(dali_sc.get_link_flags())
+        dali_include_flags = " ".join(dali_sc.get_include_flags())
+        dali_cflags = " ".join(dali_sc.get_compile_flags())
+        dali_lflags = " ".join(dali_sc.get_link_flags())
```

Comment:
Was `dali_lib_path` removed on purpose?

Reply:
Yes, it is a local variable that is not used anywhere in this funciton.